qualtran/cirq_interop/t_complexity_protocol.py

```diff
@@ -133,7 +133,7 @@ def _from_directly_countable(stc: Any) -> Optional[TComplexity]:
         quregs = get_named_qubits(stc.signature)
         qm = cirq.SimpleQubitManager()
         op, _ = stc.as_cirq_op(qubit_manager=qm, **quregs)
-        return t_complexity(cirq.decompose_once(op))
+        return t_complexity(op)
```

Comment:
Why this change?

Reply:
It was causing a failure if it encountered CRz. I don't understand why you'd want to decompose it again. `Controlled.as_cirq_op` (above) should do the trick